src/BootstrapTooltipContext.js

```diff
@@ -37,23 +49,27 @@ export default declare("BootstrapTooltip.widget.BootstrapTooltipContext", [_boot
         if (this._contextObj) {
             this.subscribe({
                 guid: this._contextObj.getGuid(),
-                callback: () => this.setTooltipTextAttribute()
+                callback: lang.hitch(this, (guid) => {
```

Comment:
What's the purpose here with `hitch` and unused `guid`? Seems like this isn't necessary.

Reply:
https://dojotoolkit.org/reference-guide/1.7/dojo/hitch.html

Reply:
Arguments removed